glide.yaml

```diff
@@ -57,3 +57,6 @@ import:
 # matches openshift/origin
 - package: github.com/gonum/graph
   version: 50b27dea7ebbfb052dfaf91681afc6fde28d8796
+
+- package: github.com/vishvananda/netlink
```

Comment:
This dependency really doesn’t belong in library-go.  Pulling this dependency into library go to reuse a 20 line function is overkill